src/libraries/Microsoft.PowerFx.Connectors/Microsoft.PowerFx.Connectors.csproj

```diff
@@ -10,6 +10,7 @@
   <!-- Nuget Properties -->
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
```

Comment:
This is a common property, can we just put it in the top level Directory.Build.props?

Reply:
Done